aws/aws-toolkit-vscode
# Lambda delete confirm 219

Added confirmation when deleting Lambda



- [ ] Infrastructure (change which improves the lifecycle management (CI/CD, build, package, deploy, lint, etc) of the application, but does not change functionality.)
- [x] Technical debt (change which improves the maintainability of the codebase, but does not change functionality)
- [ ] Testing (change which modifies or adds test coverage, but does not change functionality)
- [ ] Documentation (change which modifies or adds documentation, but does not change functionality)

## Description

* Added confirmation when deleting Lambda
* Removed use of globals from deleteLambda

## Motivation and Context

In response to issue.

## Related Issue(s)

https://github.com/aws/aws-toolkit-vscode/issues/219

## Testing

Updated test to:
* Check that delete is called when confirmed
* Check that delete is not called when cancelled
* Refresh is called on both confirm and cancel

## Screenshots (if appropriate)

## Checklist





## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
